projects/transformers/callbacks/sparsity.py

```diff
@@ -19,10 +19,20 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
+import wandb
 from transformers import TrainerCallback
+from transformers.utils import logging
 
+from nupic.research.frameworks.pytorch.model_utils import count_nonzero_params
 from nupic.torch.modules import SparseWeights, rezero_weights
 
+logger = logging.get_logger(__name__)
+
+__all__ = [
+    "RezeroWeightsCallback",
+    "SparsifyFCLayersCallback",
+]
+
 
 class SparsifyFCLayersCallback(TrainerCallback):
```

Comment:
Should we keep this? I am ok with removing it if we have a better way not, that can avoid confusion.

Reply:
I can remove it, but we'll need to update the `sparse_bert_100k` to use one of the new models. The architecture will be the same, just different functionality will load it.

Reply:
Your call if it is worth the trouble or not. If we don't do it we can at least add a note to the docstring saying it is deprecated.